apps/multisig/config.ts

```diff
@@ -28,7 +28,7 @@ export const SUPPORTED_CHAINS: Record<string, SupportedChain> = {
   'bittensor': {
     id: 'bittensor',
     subscanUrl: 'https://bittensor.com/scan',
-    rpcs: [{ url: `wss://entrypoint-finney.opentensor.ai/` }],
+    rpcs: [{ url: 'wss://bittensor-finney.api.onfinality.io/ws?apikey=' }],
```

Comment:
?apikey= ?

Reply:
That's on purpose, see:

https://github.com/TalismanSociety/signet-web/pull/62/files#diff-ce16d69fc4eff930578ef646aa1db499d39f3ada97148a1b9526f128c6d45907R37-R39